docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,170 @@
+# Bitbucket integration
+
+!!! note
+    (CUSTOMER FACING MESSAGE NEEDED) We have launched Bitbucket integration to receive your feedback. Some metrics are not available yet.
+
+Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
+
+-   [Deployment frequency](../metrics/accelerate.md#deployment-frequency)
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App** and follow the instructions on the Bitbucket UI to install the app on your account.
+
+    (IMAGE)
+
+1.  Wait until you get a confirmation that Pulse successfully created the integration and the webhook on Bitbucket.
+            
+    (IMAGE)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Choose the workspace that you want to connect the integration. You will only be able to select a workspace that you are the owner.
+
+    (IMAGE)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Click **Complete setup**.
+
+## Automatic deployment detection using merged pull requests {: id="deployment-detection-merged-pr"}
+
+!!! note
+    Pulse will use your merged pull requests to automatically detect your deployments. Other strategies to detect deployments will be available with a future version of Pulse.
+
+-   Pulse considers a deployment every merged pull request that **targets the default branch** of the repository.
+-   The deployment date is the timestamp when the corresponding pull request is merged.
+-   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you [squash or fast-forward the commits when merging the pull request](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Merge-strategies), since Pulse processes all the original commits before any changes to the Git history.
+-   Pulse associates all Bitbucket teams of the author of a merged pull request with the corresponding deployment, excluding teams with less than two members. Pulse only takes changes to Bitbucket teams into account on pull requests merged after those changes.
```

Comment:
```suggestion
-   Pulse associates all Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding user groups with less than two members. Pulse only takes changes to Bitbucket teams into account on pull requests merged after those changes.
```
I believe BB concept of teams is called user groups. @claudiacodacy can confirm that
I'm not sure which one you'd like to use here

Reply:
Yep, you're right. It seems that in Bitbucket you can have [2 types of accounts: Individuals and Team accounts](https://community.atlassian.com/t5/Bitbucket-questions/Difference-between-teams-and-groups/qaq-p/755346), but they use [User groups to organize members of a workspace](https://support.atlassian.com/bitbucket-cloud/docs/organize-workspace-members-into-groups/). 👍